modules/games/definitions/neutreeko/meta.ts

```diff
@@ -0,0 +1,15 @@
+import { NeutreekoDefinition } from "./_types";
+
+const neutreekoMeta: NeutreekoDefinition["meta"] = {
+  id: "neutreeko",
+  author: "Jan Kristian Haugland",
+  name: "Neutreeko",
+  tags: [],
```

Comment:
```suggestion
  tags: ["formation", "iggamecenter", "slide"]
```

* `formation` is for games where the win condition is to arrange pieces in a formation
* `iggamecenter` is for games implemented on that site
* `slide` is for games where we have to move as far in a direction as possible